ledger-core/v2/application/testwalletapi/handlers.go

```diff
@@ -139,23 +139,32 @@ func (s *TestWalletServer) Transfer(w http.ResponseWriter, req *http.Request) {
 		TraceID: traceID,
 		Error:   "",
 	}
-	defer func() { s.mustWriteResult(w, result) }()
+	defer func() {
+		if len(result.Error) != 0 {
+			logger.Error(result.Error)
+		}
+		s.mustWriteResult(w, result)
+	}()
 
 	fromRef, err := insolar.NewReferenceFromString(params.From)
 	if err != nil {
-		result.Error = err.Error()
+		result.Error = throw.W(err,
+			fmt.Sprintf("Failed to create reference from string (%s), sender reference", params.From), nil,
```

Comment:
better logging usage: it's not very clear what usage is for, but as far as I see it should be like that
```
 		result.Error = throw.W(err,  "failted to create reference from string", struct{ From string }{ From: params.From }
```

Reply:
Fixed)